[BRS-517] - Add Disaster Recovery Script

davidclaveau opened the pull request:
### Ticket:
BRS-517

### Ticket URL:
[#517](https://github.com/bcgov/parks-data-register/issues/517)

### Description:
- Add Disaster Recovery script to AR
- The Disaster Recovery script will initiate the steps to restore, backup, and/or delete a table in DynamoDB
- Refer to the [README](https://github.com/bcgov/bcparks-ar-api/blob/6920a114ed135418febf9f43d5aea2a7d51197ec/arSam/disaster-recovery/README.md) for more information regarding the script processes.

marklise commented:
In `functions.js` you have a lot of try-catch that simply re-throws the error to the caller. It might be a bit redundant to do that, and perhaps better to eliminate `functions.js` and to embed the logic in one level up in the hierarchy.

davidclaveau commented:
Addressed in commit `c7fd66`. Updated the script's files to streamline functionality and reduce the number of try/catches in `function.js`. Also added some `config.js` validation checks and updated `README.md`.

cameronpettit reviewed (approved):
Nice work on the README.
